go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containers/conmon-rs
 
-go 1.18
+go 1.20
```

Comment:
required by `errors.Join`, but we already use go 1.20 in CI so I assume this is fine :shrug: